tests/kuttl/test-suites/default/watcher-api/03-deploy-watcher-api.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 type: Opaque
 stringData:
   WatcherPassword: password
+  transport_url: rabbitmq-transport-url-watcher-kuttl-watcher-transport
```

Comment:
sorry for being picky but I feel this example may be confusing although it works fine for the test. This has the name of the transporturl secret while the expected content of this key is the value for transport_url which is really a data which is inside it. It would be something like:

transport_url: rabbit://default_user:password@rabbitmq.watcher-kuttl-default.svc:5671/?ssl=1

Reply:
/lgtm

Reply:
that is a good point to clarify, this value as of now is meaningless because we don't have any easy way to check the generated config, I just needed the secret with the field to be there. I'll change it to make it a dummy url so it's clearer what the content should be until we have the secret created by the Watcher controller

Reply:
Actually, I think in the way you have defined the template parameters in watcherapi_controller and config file, the expected value is the url, not the secret name. Other operators are doing a similar approach, so I'd go with it

Reply:
you're correct, I meant to say that the kuttl tests require a secret containing some `transport_url` , but they don't actually enforce that the value is correct atm, we'll do that once we create the deployment